win32ss/gdi/ntgdi/region.c

```diff
@@ -2149,7 +2150,6 @@ REGION_bXformRgn(
             /* Loop all rects in the region */
             for (i = 0; i < prgn->rdh.nCount - 1; i++)
             {
-                NT_ASSERT(prgn->Buffer[i].top <= prgn->Buffer[i].bottom);
```

Comment:
This assert is there for a reason!

Reply:
Well, that reason is not obvious to me: again, why is it correct not to check the last rectangle (anymore)?